prepare-chroot-builder

```diff
@@ -112,6 +112,9 @@ if [ -n "$USE_QUBES_REPO_VERSION" ]; then
     if [ "$DIST" = "$DIST_DOM0" ]; then
         cp ${PLUGIN_DIR}/repos/qubes-repo-dom0-fedora.repo $DIR/etc/yum.repos.d/
     fi
+    if [ "x$QUBES_MIRROR" != "x" ]; then
+        sed -i "s#http://yum.qubes-os.org#$QUBES_MIRROR#" $DIR/etc/yum.repos.d/qubes-repo-*.repo
```

Comment:
What about https? Even if not used here right now, switching to it later will break QUBES_MIRROR setting.